frontend/system/src/Admin/EditDrug/DrugEditForm/DrugEditForm.tsx

```diff
@@ -137,11 +61,8 @@ export const DrugEditForm = ({
       },
     })
       .then(() => {
-        invalidateQueries();
-        setIsSaving(false);
-
         if (!initialEntity) {
-          // new entity - clear the form and show success snack
+          // new entity - clear the form
           success(
```

Comment:
It feels like we should possibly navigate somewhere on submit, Feels like of weird to be staring at the same blank form after submitting.
Probably quite good from a UX though in terms of being able to quickly submit a bunch quickly, so on the fence...

Reply:
Yeah that was my thinking, will be good to hear what Louisa has to say, what the more common use case would be... possibly we could do some kind of Add/Add & Next buttons thing?